### PR TITLE
Improve Industrialcraft2 plugin

### PR DESCRIPTION
**Pull request by temp1011**
Requires https://github.com/ForestryMC/ForestryMC/pull/2050.

Some small fixes to `PluginIC2` to let rubber trees work in the normal tree farm again and clean up backpacks and crates a bit.

### CODE REVIEW

**Review by Nedelosk — CHANGES_REQUESTED** [review]
Could you please use the new Module Helper that I added with the last commit.

**Comment by temp1011** [comment]
Now uses `ModuleHelper`

**Comment by temp1011** [comment]
Would be nice if I could improve this https://github.com/ForestryMC/ForestryMC/blob/24c8add7fcba5b3067eff8244ef4b0442658460d/src/main/java/forestry/farming/logic/crops/CropRubber.java#L43-L59 Are the IC2 people good at adding to the API is people ask them too?

**Comment by temp1011** [comment]
This now fixes #2139.

File: src/main/java/forestry/plugins/PluginIC2.java [hunk 1]
```diff
@@ -147,12 +144,10 @@ public void preInit() {
...
+		if (Recipes.recyclerBlacklist != null && ModuleHelper.isEnabled(ForestryModuleUids.APICULTURE)) {
 			ItemRegistryApiculture beeItems = ModuleApiculture.getItems();
-			if (beeItems != null) {
-				Recipes.recyclerBlacklist.add(Recipes.inputFactory.forStack(new ItemStack(beeItems.beeQueenGE)));
-				Recipes.recyclerBlacklist.add(Recipes.inputFactory.forStack(new ItemStack(beeItems.beePrincessGE)));
-			}
+			Recipes.recyclerBlacklist.add(Recipes.inputFactory.forStack(new ItemStack(beeItems.beeQueenGE)));
+			Recipes.recyclerBlacklist.add(Recipes.inputFactory.forStack(new ItemStack(beeItems.beePrincessGE)));
 		} else {
 			Log.error("IC2 Recipes.recyclerBlacklist not found.");
```
Comment (mezz) [inline src/main/java/forestry/plugins/PluginIC2.java]: This will log if apiculture is disabled, there should be no log in that case.

**Review by mezz — CHANGES_REQUESTED** [review]
Looks good, just one minor thing.

**Review by mezz — APPROVED** [review]

**Pull request merged**